### PR TITLE
doc/user: add release notes for v0.26.1

### PR DESCRIPTION
**Pull request by benesch**
Just a draft for now. We may need to add another change to v0.26.1.

### Motivation

* Weekly release notes PR.

### Testing

- [x] This PR has adequate test coverage / QA involvement has been duly considered.

### Release notes

This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):

  - n/a

### CODE REVIEW

**Review by morsapaes — APPROVED** [review]

File: doc/user/content/release-notes.md [hunk 1]
```diff
@@ -21,12 +21,28 @@ the PR description instead. They will be migrated here during the release
 process by the release notes team.
 {{< /comment >}}
 
-{{% version-header v0.26.0 %}}
+{{% version-header v0.26.1 %}}
 
-v0.26.0 is a **long-term support (LTS)** release of Materialize. We will support
```
Comment (andrioni) [inline doc/user/content/release-notes.md:26]: Should we maybe say that the `0.26.x` series of versions are LTS, and only the latest of them is supported at any point in time?

Comment (andrioni) [reply doc/user/content/release-notes.md:26]: I guess my point here is clarifying that the 0.26.x family of releases is what's being supported in the long term, not each individual 0.26 release. Nitpick, anyway!

Comment (umanwizard) [reply doc/user/content/release-notes.md:26]: I agree with Alessandro

Comment (morsapaes) [reply doc/user/content/release-notes.md:26]: Tweaked it slightly, added that bit as a note and got a 👍 from @umanwizard.

**Pull request merged**